### DISCUSSION TITLE
customization pack plasma/kde deepin style

### DISCUSSION THREAD

**Discussion by edoardge** [linuxdeepin/developer-center#2960] [Features Request & Ideas | 特性请求 & 头脑风暴]
Could you create a customization pack for plasma/kde (plasma theme, aurorae/kwin) with deepin and deepin dark style...??? Thank you... This can be complemented by deepin gtk theme... Thank you again...

**Comment by justforlxz** [comment]
Sorry, this topic will be closed soon. If it is necessary to discuss it again, please create a new topic.

**Comment by BLumia** [comment]
Actually, you don't need a theme pack specific for KDE. Qt based app can will use Qt platform theme and Deepin provide a theme called chameleon which can be used by passing `-platformtheme deepin -style chameleon` to the command line or setting the corresponding environment variables.

But since chameleon is designed sort-of for touch screen and it's problematic to apply to all Qt apps in the current state, so, I actually don't suggest doing that.
